integration/k8s_custom_deploy/Tiltfile

```diff
@@ -1,15 +1,15 @@
 
 include('../Tiltfile')
 
-enable_feature('k8s_custom_deploy')
-
 k8s_custom_deploy(
     'custom-deploy-int-test',
     # forcibly delete and re-create the deployment whenever run
-    cmd='kubectl delete --ignore-not-found=true --wait=true -f "${DEPLOY_FILE}" >/dev/null && kubectl create -oyaml -f "${DEPLOY_FILE}"',
+    deploy_cmd='kubectl delete --ignore-not-found=true --wait=true -f "${DEPLOY_FILE}" >/dev/null && kubectl create -oyaml -f "${DEPLOY_FILE}"',
```

Comment:
i wonder if we should call this `apply` instead of `delete` to better match the `kubectl` verbs...

Reply:
Renamed everywhere in e4adb2efb to be `apply_cmd` in Starlark and `ApplyCmd` in API